dnf-behave-tests/dnf/environment.py

```diff
@@ -211,6 +215,37 @@ def get_microdnf_cmd(self, context):
 
         return result
 
+    def get_dnf5_cmd(self, context):
+        if context.dnf.dnfdaemon_mode:
+            result = ["dnfdaemon-client"]
```

Comment:
This breaks `--command` option in dnfdaemon_mode. I'd prefer handle it consistently with microdnf5 and rely on `--command=dnfdaemon-client`.